src/api/goods.ts

```diff
@@ -10,10 +10,11 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    // eslint-disable-next-line max-len
+    .then(goods => goods.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5)); // sort and get the first 5
```

Comment:
Remove the eslint-disable comment and break the line into multiple lines to improve readability.